models/reviews.models.js

```diff
@@ -0,0 +1,31 @@
+const db = require("../db/connection");
+
+exports.fetchReviewById = (review_id) => {
+  return db
+    .query(
+      `
+    SELECT * FROM reviews WHERE review_id = $1;
+    `,
+      [review_id]
+    )
+    .then((result) => {
+      if (result.rowCount === 0) {
+        return Promise.reject({ msg: "ID Not Found", status: 404 });
+      } else {
+        return result.rows[0];
+      }
+    });
+};
+
+exports.fetchAllReviews = () => {
+  return db
+    .query(
+      `
+    SELECT * FROM reviews
```

Comment:
Need to adjust this so that you can also get back the comment count

comment_count which is the total count of all the comments with this review_id - you should make use of queries to the database in order to achieve this.